thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -284,26 +292,88 @@ export class AccountManager {
             "message": ""
         }
 
-        return await fetch("/api/auth/logout", {
+        const response: Response = await fetch("/api/auth/logout", {
             method: "POST",
             headers: {
                 "Content-Type": "application/json"
             },
             credentials: "include"
-        }).then((response) => {
-            if (response.status === STATUS.RESET_CONTENT) {
-                this.token = "";
-                this.expiresAt = 0;
-            } else {
-                httpResponse.result = false;
-                httpResponse.message = "Unexpected return code from server: " + response.status;
-            }
-            return httpResponse;
-        }).catch((error) => {
-            console.log("Failed to logout user, error is: " + error);
+        });
+
+        if (response.status === STATUS.RESET_CONTENT) {
+            this.token = "";
+            this.expiresAt = 0;
+        } else {
+            console.log("Unexpected return code from server: " + response.status);
+            httpResponse.result = false;
+            httpResponse.message = "Request failed, try again.";
+        }
+        return httpResponse;
+    }
+
+    /**
+     * Creates a game made by the current user.
+     */
+    public async createGame(): Promise<HttpResponse> {
+        const httpResponse: HttpResponse = {
+            "result": true,
+            "message": ""
+        }
+
+        const response: Response = await fetch("/api/add/game", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                "Authorization": "Basic " + this.token,
+            },
+            credentials: "include"
```

Comment:
We don't need to include credentials here. That will send the HttpOnly refresh token, which isn't used by this endpoint.